test/integration/IntegrationTest.cpp

```diff
@@ -39,35 +39,35 @@
 IntegrationTest::IntegrationTest()
     : _sendAllocator(memory::packetPoolSize, "IntegrationTest"),
       _audioAllocator(memory::packetPoolSize, "IntegrationTestAudio"),
-      _jobManager(std::make_unique<jobmanager::JobManager>()),
       _mainPoolAllocator(std::make_unique<memory::PacketPoolAllocator>(4096, "testMain")),
       _sslDtls(nullptr),
       _network(transport::createRtcePoll()),
       _pacer(10 * utils::Time::ms),
       _instanceCounter(0),
-      _numWorkerThreads(getNumWorkerThreads())
+      _numWorkerThreads(2 * getNumWorkerThreads())
 {
 }
 
+// TimeTurner time source must be set before starting any threads.
+// Fake internet thread, JobManager timer thread, worker threads.
 void IntegrationTest::SetUp()
 {
-
     if (__has_feature(address_sanitizer) || __has_feature(thread_sanitizer))
     {
-        GTEST_SKIP();
+        // GTEST_SKIP();
```

Comment:
This will be disabled again until we find a reliable way of running in Jenkins.

Reply:
Shall we change it back/disable before merging?